frontend/src/components/Dashboard/CrossingDetailPage/CrossingStatusHistory.js

```diff
@@ -1,19 +1,82 @@
 import React, { Component } from 'react';
+import { get } from 'lodash';
+import classnames from 'classnames';
+import { ContainerQuery } from 'react-container-query';
+import { displayedInputs, statusIcons } from 'constants/StatusConstants';
+import { LARGE_ITEM_MIN_WIDTH } from 'constants/containerQueryConstants';
+import DateTime from 'components/Dashboard/CrossingListPage/CrossingListItem/DateTime';
+import 'components/Dashboard/CrossingDetailPage/CrossingStatusHistory.css';
+
+const containerQuery = {
+  'CrossingStatusHistory--lg' : {
+    minWidth: LARGE_ITEM_MIN_WIDTH,
+  }
+}
 
 class CrossingStatusHistory extends Component {
   render() {
     const { history } = this.props;
 
     return (
-      <div>
-        <div>Crossing Status History: {this.props.crossingId}</div>
-        {history.map(update => 
-          <div> Update: {update.id} </div>
+      <ContainerQuery query={containerQuery}>
+        {(params) => (
+          <div className={classnames(params, "CrossingStatusHistory")}>
+            <h2 className="CrossingStatusHistory__section-header">
+              Crossing History
+            </h2>
+            <div className="CrossingStatusHistory__list-wrapper">
+              {history.map((update, i) => {
+                const user = get(update, 'userByCreatorId', {});
```

Comment:
I haven't used get like this before, just curious, what are the benefits?

Reply:
This was a pattern that @ifsimicoded suggested. At the top, we are able to import a single function from the lodash library. The `get` function can traverse a nested object  but the real benefit is that it comes back with an assignable default value instead of bomb.